bindings/python/tests/session_test.py

```diff
@@ -957,9 +957,14 @@ def test_deprecated(self) -> None:
         with self.assertWarns(DeprecationWarning):
             self.session.set_dht_settings(dht_settings)
         with self.assertWarns(DeprecationWarning):
-            self.session.dht_get_mutable_item("a" * 32, "salt")
+            self.session.dht_get_mutable_item("a" * 32, "salt")  # type: ignore
         with self.assertWarns(DeprecationWarning):
-            self.session.dht_put_mutable_item("a" * 64, "b" * 32, "data", "salt")
+            self.session.dht_put_mutable_item(
+                "a" * 64,  # type: ignore
+                "b" * 32,  # type: ignore
+                "data",  # type: ignore
+                "salt",  # type: ignore
```

Comment:
I suppose all of these should technically be `bytes`, right?

Reply:
Right, the type stubs declare these ase `bytes`, but I wanted to add a test with `str` as a deprecated usage.